www/src/data/sidebars/doc-links.yaml

```diff
@@ -496,6 +496,9 @@
             - title: Optimizing Site Performance with Guess.js
               link: /docs/optimizing-site-performance-with-guessjs/
               breadcrumbTitle: Guess.js
+            - title: Profiling Site Performance with React Profiler
+              link: /docs/profiling-site-performance-with-react-profiler/
+              breadcrumbTitle: React Profiler
```

Comment:
@pieh what does the breadcrumbTitle do? Wasn't sure.

Reply:
![Screenshot 2020-03-02 at 19 43 57](https://user-images.githubusercontent.com/419821/75706906-3c187980-5cbe-11ea-93a3-56d5052925b6.png)

That's optional, and used to put a title in the "breadcrumbs" above article (useful when full article name is long and would made breadcrumbs spill over single line). If it's not defined, we will use regular title, but in this case I think your setup is actually correct (it would be too long otherwise)